### DISCUSSION TITLE
lite-xl

### DISCUSSION THREAD

**Discussion by rawhuul** [catppuccin/catppuccin#2008] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Dear Catppuccin Themes team,

I am a user of the Lite-xl text editor and I would like to express my appreciation for the beautiful themes that you have created for various text editors. I am particularly fond of your Catppuccin themes and would love to be able to use them in Lite-xl.

[Lite-xl](https://github.com/lite-xl/lite-xl) is a lightweight and fast text editor that has gained popularity among developers due to its simplicity and efficiency. However, its limited set of built-in themes may not be sufficient for some users, and they may want to use themes from other sources.

As a user of Lite-xl, I would like to request that you make your Catppuccin themes available for use in Lite-xl. Your themes are not only visually appealing but also provide a comfortable and pleasant working environment, and I believe that many Lite-xl users would appreciate having them as an option.

I understand that creating themes for different text editors may require some effort and resources, but I hope that you will consider this request as it would greatly benefit Lite-xl users and enhance their experience with your beautiful themes.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by justTOBBI** [comment]
🤓 well akshually its catppuccin 
But thanks for the nice paragraph

**Reply by rawhuul** [reply]
> 🤓 well akshually its catppuccin But thanks for the nice paragraph

Sorry for the typo 😛

**Reply by justTOBBI** [reply]
@basicfunc Also did you mean to click the 'Yes, I'm willing to make the port' Button?
